app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -10,28 +10,49 @@ import Status from '../components/Status';
 import { connect } from 'react-redux';
 import { getAnswerForQuestionAndBuilding } from '../selectors/answersSelector';
 import { getDependentQuestionsForOptions } from '../selectors/questionsSelector';
-import { createAnswer, updateAnswer } from '../actions/answers';
+import { createAnswer, updateAnswer, updateLocalAnswer } from '../actions/answers';
 
 
 class OptionsContainer extends React.Component {
   /**
-   * Callback for when an option has been triggered. Will update the answer in database and store.
-   *
-   * @param option_id   id of the selected option
-   * @param value       The data of the updated answer. For range options, this is the number the user inputted;
-   *                      for dropdown options, it's the text of the option that was selected.
+   * Returns answer data in the format expected for a fetch request.
    */
-  handleSelect(option_id, value) {
-    // Set up answer data to send in fetch request
-    const answer = {
+  getAnswerData(option_id, value) {
```

Comment:
Since this doesn't affect the return object's keys, can we camel-case `option_id`? We will address making everything in camel case (up until what we pass to our fetch functions) for our JS later.